File: app/src/main/java/model/Forecast.java
package model;

import java.util.List;

/**
 * Created by gjacobs on 10/31/15.
 */
public class Forecast {
    City city;
    String cod;
    float message;
    int cnt;
    List<ForecastDetails> list;

    public City getCity() {
        return city;
    }

    public void setCity(City city) {
        this.city = city;
    }

    public String getCod() {
        return cod;
    }

    public void setCod(String cod) {
        this.cod = cod;
    }

    public float getMessage() {
        return message;
    }

    public void setMessage(float message) {
        this.message = message;
    }

    public int getCnt() {
        return cnt;
    }

    public void setCnt(int cnt) {
        this.cnt = cnt;
    }

    public List<ForecastDetails> getList() {
        return list;
    }

    public void setList(List<ForecastDetails> list) {
        this.list = list;
    }
}

File: app/src/main/java/model/Forecast.kt
package model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Embedded
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

/**
 * Created by gjacobs on 10/31/15.
 */
@Entity(tableName = "forecast", primaryKeys = arrayOf("lattitude", "longitude"))
data class Forecast(val void: Unit? = null) {
    var lattitude: Double = 0.0
    var longitude: Double = 0.0
    @Embedded
    lateinit var city: City
    lateinit var cod: String
    var message: Float = 0.0F
    var cnt: Int = 0
    lateinit var list: ArrayList<ForecastDetails>
}
